raycicmd/step_node.go

```diff
@@ -38,7 +38,10 @@ type stepNode struct {
 	depSet        map[string]struct{}
 	reverseDepSet map[string]struct{}
 
-	// Marked is set to true when the node will be included in a conversion.
+	// rejected means that this step will not be included in the conversion.
```

Comment:
I'm wondering if we should perhaps set default value for these variables? Are they always assigned a value when constructed?

Reply:
it does not have a constructor, it is taking zero value as default value.